osu.Game/Online/Multiplayer/GameplayStartCountdown.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using MessagePack;
+
+namespace osu.Game.Online.Multiplayer
+{
+    /// <summary>
+    /// A <see cref="MultiplayerCountdown"/> indicating that gameplay will start after a given period of time.
+    /// </summary>
+    [MessagePackObject]
+    public class GameplayStartCountdown : MultiplayerCountdown
```

Comment:
This reads a touch ambiguously, especially when it exists next to another countdown called `MatchStartCountdown` (which feels almost synonymous?).

How about:

```suggestion
    /// <summary>
    /// A <see cref="MultiplayerCountdown"/> started by the server when clients being to load.
    /// Indicates how long until gameplay will forcefully start, excluding any users which have not completed loading,
    /// and forcing progression of any clients that are blocking load due to user interaction.
    /// </summary>
    [MessagePackObject]
    public class ForceGameplayStartCountdown : MultiplayerCountdown
```